portal/src/views/shared/SidebarNav.vue

```diff
@@ -105,6 +105,15 @@ export default Vue.extend({
             },
         };
     },
+    computed: {
```

Comment:
Anyway to do this in CSS?

Reply:
The only way that I see with css only would be to add 2 <i> elements, 1 for fieldkit, 1 for floodnet and hide/show them via display css property when the body has / doesn't have the floodnet class. But I can't say that's much nicer.

Another solution that might be nicer would be to set a floodnet state to true in App.vue and check for that. 

What would you prefer?

Reply:
For the icons, for now what if we created a utility function that builds the icon name/path by convention. So we call the function return partnerIcon("icon-logo-%s") and that interpolates the brand/partner name into the string. This would also work for i18n keys. This way the test against the document classes is only done once and we don't have conditionals sprinkled around.

Reply:
I've pushed this function. Please take a look where I placed it (a bit unsure that's the right place)